### PR TITLE
Weather Application

### PR DESCRIPTION
**Pull request by biruk-bereka**
## In this PR I:
  - Set  up project with create-react-app
  - Create home and details routes 
  - Fetch weather data from openWeatherMap end point
  - Display weather data in home component and weather details in details component 
  - Implement search functionality
  - Add test case for home and details component

[Presentation Video](https://www.loom.com/share/db10f4ad122943ceaaa276af3b4e2fc6)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1 +1,227 @@
...
+If you like this project give a star!! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
```
Comment (ghost) [inline Readme.md]: - [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your `Acknowledgement` section of your readme file

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @biruk-bereka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ All tests are passing
✅ App is designed as expected 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Readme.md (hunk 1 above)
Comment (biruk-bereka) [reply Readme.md]: @frank1738 Thank you for your feedback.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @biruk-bereka 👋, 

Well done for implementing the required change 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**